Gruntfile.js

```diff
@@ -166,8 +70,11 @@ module.exports = function(grunt) {
             js: {
                 // Files to be uploaded.
                 upload: [{
-                    src: 'dist/smooch.min.js',
+                    src: 'dist/smooch.js',
                     dest: 'smooch.min.js'
+                },{
+                    src: 'dist/smooch.js.map',
```

Comment:
Lol I had already started looking a week ago but then squirrel.

When would these source maps get downloaded in prod? When you open dev tools?